Contribute/content/seo-reference.md

```diff
@@ -11,30 +11,30 @@ ms.custom: external-contributor-guide
 
 # SEO quick reference guide
 
-Use these search-engine optimization (SEO) tips as a quick reference to the basics of on-page SEO for contributors to Microsoft Learn. This guide can be an effective tool in optimizing your content once you've done your keyword research and aligned your content with user intent. Adhering to these SEO best practices can improve your overall content quality and discoverability by users, search engines, and large language models.
+Use these search engine optimization (SEO) tips as a quick reference to the basics of on-page SEO for contributors to Microsoft Learn. This guide can be an effective tool in optimizing your content once you've done your keyword research and aligned your content with user intent. Adhering to these SEO best practices can improve your overall content quality and discoverability by users, search engines, and large language models (LLMs).
 
 ## Search results: main elements
 
-- [**Metadata title:**](#metadata-titles) It has the greatest impact on search rank and click-through rate (CTR). The meta title isn't the same as the main heading (H1) on the page.
-- [**URL:**](#optimized-url) Once chosen, it shouldn't be changed because it affects tracking. Changing the URL could send conflicting signals to search engines. The URL should be concise and specific to aid search engines and improve user experience.
-- [**Metadata description:**](#metadata-description) This short block of text gives users a preview of what your pages is about. It should be a summary that entices users to click.
+- [**Meta title:**](#meta-titles) The meta title is entered in the YAML header in a markdown document (or between the head tags in HTML). It is displayed on the search engine results page (SERP) and browser tab. It is the title that describes the entire page. It has the greatest impact on search rank and click-through rate (CTR). The meta title isn't the same as the main heading (H1) on the page.
+- [**URL:**](#optimized-url) Once chosen, the URL shouldn't be changed because it affects tracking. Changing the URL could send conflicting signals to search engines. The URL should be concise and specific to aid search engines and improve user experience.
+- [**Meta description:**](#meta-description) The meta description is also entered in the YAML header in a markdown document (or between the head tags in HTML). It is displayed in the SERPs below the URL and meta title. This short block of text gives users a preview of what your pages is about. It has a positive impact on CTR and should be a summary that entices users to click.
```

Comment:
```suggestion
- [**Meta title:**](#meta-titles) The meta title is entered in the YAML header in a Markdown document (or between the head tags in HTML). It's displayed on the search engine results page (SERP) and browser tab. It's the title that describes the entire page. It has the greatest impact on search rank and click-through rate (CTR). The meta title isn't the same as the main heading (H1) on the page.
- [**URL:**](#optimized-url) Once chosen, the URL shouldn't be changed because it affects tracking. Changing the URL could send conflicting signals to search engines. The URL should be concise and specific to aid search engines and improve user experience.
- [**Meta description:**](#meta-description) The meta description is also entered in the YAML header in a Markdown document (or between the head tags in HTML). It's displayed in the SERPs below the URL and meta title. This short block of text gives users a preview of what your pages is about. It has a positive impact on CTR and should be a summary that entices users to click.
```